### PR TITLE
cleaning tool errors

### PR DESCRIPTION
**Pull request by Danilodum**
preparing for merge in master

### CODE REVIEW

File: src/main/java/com/funkydonkies/w4v3/curve/CustomCurveMesh.java [hunk 1]
```diff
@@ -67,17 +67,19 @@ public void setMeshPoints(final int i) {
...
 	public void getSegmentPoint(final int i) {
+		final int mulLength = 3, offset = 2; 
 		 
 		if (splinePoints[i].getY() < splinePoints[i + 1].getY()) {
 			final Vector3f vecNew = new Vector3f().add(splinePoints[i + 1]);
 			vecNew.setY(splinePoints[i].getY());
-			meshStructurePoints[splinePoints.length * 3 + 2 * i] = splinePoints[i];
-			meshStructurePoints[splinePoints.length * 3 + 2 * i + 1] = vecNew;		
+			meshStructurePoints[splinePoints.length * mulLength + offset * i] = splinePoints[i];
+			meshStructurePoints[splinePoints.length * mulLength + offset * i + 1] = vecNew;
```
Comment (jonathanraes) [inline src/main/java/com/funkydonkies/w4v3/curve/CustomCurveMesh.java]: looks very clean, only thing maybe add parenthesis on these kind of things to make it even more clear

Comment (Danilodum) [inline src/main/java/com/funkydonkies/w4v3/curve/CustomCurveMesh.java]: I agree

**Comment by jonathanraes** [comment]
nice looks clean

**Comment by jonathanraes** [comment]
Still some findbugs errors regarding splinepoints and curvepoints though. We're still looking for a solution on that.
Can be merged into customcurvemesh

**Pull request merged**